package.json

```diff
@@ -110,6 +110,7 @@
     "redux-thunk": "^2.1.0",
     "sass-loader": "^3.1.2",
     "style-loader": "^0.13.0",
+    "term.js": "rancher/term.js#583581949471d833849db71cc9b1d3b74d2d8474",
```

Comment:
We only rely on the client-side library, which doesn't need to be compiled and doesn't have any require/import business.

We don't use the middleware (`lib/index.js`) at all. Building this as a node module isn't necessary.  Is there a package.json way to say we just want `src/term.js` as a client dependency (like a bower dependency in this case) and ignore that this thing has anything to do with nodejs?